### ISSUE TITLE
Turtle exercises

### ISSUE THREAD

**Issue by gorelhedin** [lunduniversity/schoolprog#24]
I have updated the `turtle` exercise:

- [x] Written it in Swedish - moved the old text to `README_EN.md`
- [x] Added subexercises/skeletons for "SARA": Sequence, Alternative, Repetition, Abstraction.
- [x] Added headings for exercises for Tårta and Blomma. Which would be nice to have before starting the Piece-of-cake exercise
- [x] Added headings for exercises for Rita koordinataxlar. Which would be nice to have before starting the Graphing exercise.

Things left to do:

- [x] Added turtle cheatsheets for both Python and Scala.
- [x] Complete the `turtle` exercise for Python. (Reasonably complete. Could add some more.)
- [ ] Translate the `turtle` exercise to Scala.
- [ ] Complete the english version.
- [ ] Translate Björn's Kojo-exercises. Put in separate exercise.

@obakanue , will you take it from here?

**Comment by gorelhedin** [comment]
@obakanue I have updated the Piece-of-cake exercise now so that it is independent of the Turtle exercise (I give them code for drawing filled rectangles). So you don't have to make the Tårta and Blomma exercises compatible.

**Comment by gorelhedin** [comment]
@obakanue Have you started working on the Turtle exercise yet? If not, I have some time this evening to work on it. Would be good to have for tomorrow.

**Comment by gorelhedin** [comment]
Closing this. Remaining things are not urgent.
